descope/auth/auth.go

```diff
@@ -126,6 +126,30 @@ func (auth *authenticationService) LogoutWithOptions(request *http.Request, opti
 	return nil
 }
 
+func (auth *authenticationService) Me(request *http.Request) (*UserResponse, error) {
+	if request == nil {
+		return nil, errors.MissingRequestError
+	}
+
+	_, refreshToken := provideTokens(request)
+	if refreshToken == "" {
+		logger.LogDebug("unable to find tokens from cookies")
```

Comment:
Not only from cookies, also from header
And yes, I know it is a copy paste, please change the source as well :)

Reply:
I don't think we take `refreshToken` from headers?

Reply:
You are right